packages/iris-grid/src/CommonTypes.tsx

```diff
@@ -1,9 +1,64 @@
 import { ModelIndex } from '@deephaven/grid';
+import {
+  TotalsTableConfig,
+  FilterCondition,
+  Format,
+} from '@deephaven/jsapi-shim';
+import { Shortcut } from '@deephaven/components';
+import { IconDefinition } from '@deephaven/icons';
 import AggregationOperation from './sidebar/aggregations/AggregationOperation';
-import { AdvancedFilter, QuickFilter } from './IrisGrid';
+import { AdvancedFilterOptions } from './AdvancedFilterCreator';
+import { OptionType } from './sidebar';
```

Comment:
As mentioned in my previous comment, I think `AdvancedFilterOptions` and `AggregationOperation` should be in here as well. Should bring `OptionType` in too, so this file doesn't have any dependencies on other files within `iris-grid`: https://github.com/deephaven/web-client-ui/pull/605#discussion_r902761491